### ISSUE TITLE
Game is way too dark (Arch Linux 64 - intel i915 / mesa)

### ISSUE THREAD

**Issue by cbxd** [supertuxkart/stk-code#2830]
Hi all,
Supertuxkart 0.9.2 has been showing very dark graphics for about two months on my machine. My computer is running Arch Linux (64 bits) with i915 graphics and Mesa 17.0.2 (see attached file regarding glx info). The intel driver is up to date and so is STK.

See attached picture showing the game options and track brightness.

Anyone facing the same issue?

Regards,
Chris

[glx.txt](https://github.com/supertuxkart/stk-code/files/887719/glx.txt)
![image](https://cloud.githubusercontent.com/assets/24394479/24579722/e192cf62-16fa-11e7-8a98-f7b2fa602f3c.png)
![image](https://cloud.githubusercontent.com/assets/24394479/24579733/186c6372-16fb-11e7-92e8-6f32d13f5bb8.png)

**Comment by Benau** [comment]
I think you somehow enabled legacy pipeline, can you post stdout.log of stk?

**Comment by deveee** [comment]
You can remove version number from FramebufferSRGBCapable in graphical_restrictions.xml. It was set to 16.0 IIRC?

**Comment by cbxd** [comment]
Hello Benau and deveee, thanks for your very quick replies.
I have just removed the version number for FramebufferSRGBCapable. It was indeed set to <16.0.
Now the brightness is back to usual.

Thank you very much!
Regards,
Chris

**Comment by Benau** [comment]
Good, i should speed up my code for a new release

**Comment by deveee** [comment]
Just to make it clear, I already removed the version number from graphical restrictions some time ago:
https://github.com/supertuxkart/stk-code/commit/7e4dfdb97e5df75585ba83945e806096c7d74cfd

I reported it to intel rather long time ago and I though that they will fix it. IIRC they made a fix, but they eventually reverted it, because it broke something else.

And actually STK can work without srgb framebuffer, which would allow to remove this hack (GLES renderer already works in this way, though I don't want to introduce any regressions, so I have to look at this much more carefully).
